prac_05/emails.py

```diff
@@ -0,0 +1,30 @@
+"""
+Emails
+Estimate: 30 minutes
+Actual:   20 minutes
+"""
+
+
+def main():
+    email = input("Email: ")
+    print(email)
+    name_and_email = {}
+    while email != "":
+        extract_name(email)
+        name_and_email[str(email).strip("[ ]' ' ")] = name
+        email = input("Email: ")
+    for item in name_and_email:
+        print(f"{name_and_email[item]} ({item})")
+
+
+def extract_name(email):
+    name = email.replace('.', ' ').split("@")
+    name.pop()
```

Comment:
A comment explaining this is to discard the unwanted info (my.jcu.edu.au, etc) might be helpful?